internal/runners/use/use.go

```diff
@@ -138,9 +140,11 @@ func (u *Use) Run(params *Params) error {
 		return locale.WrapError(err, "err_use_default", "Could not configure your project as the global default.")
 	}
 
-	u.out.Print(fmt.Sprintf("[NOTICE]%s[/RESET] [ACTIONABLE]%s[/RESET]",
+	u.out.Print(fmt.Sprintf("[NOTICE]%s[/RESET] [ACTIONABLE]%s[/RESET] %s [ACTIONABLE]%s[/RESET]",
 		locale.Tl("use_notice_switched_to", "Switched to"),
-		params.Namespace.Project),
+		params.Namespace.Project,
+		locale.Tl("use_notice_located_at", "located at"),
+		setup.ExecDir(projectTarget.Dir())),
```

Comment:
I think this can be accomplished without the `fmt.Sprintf`. The locale string would be something like:
`"[NOTICE]{{.V0}}[/RESET] [ACTIONABLE]{{.V1}}[/RESET]{{.V2}}[ACTIONABLE]{{.V3}}[/RESET]"`

You can find examples of this in our `en-us.yaml` file